index.html

```diff
@@ -41,5 +41,19 @@
     </a>
 </div>
 </div>
-    </body>
+
+<div class="ui divider"></div>
+<div class="ui grid">
```

Comment:
Can you make the grid `stackable`, `<div class='ui stackable grid>` this will make it responsive out of the box for us, with that it might be worth switching the side the twitter feed is on to the right-hand side, as the way it is now, if someone viewed this on a phone it would show the twitter feed first over the body of the page, by moving the twitter feed to the right-hand side will fix this.